tests/test_compat.py

```diff
@@ -3,6 +3,12 @@
 from datetime import datetime
 from typing import Dict, Generic, List, NewType, Optional, Set, Tuple, TypeVar, Union
 
+if sys.version_info[:2] == (3, 7):
+    from typing_extensions import Literal
+else:
+    from typing import Literal
```

Comment:
You can import `Literal` from `serde.compat`  👍 

https://github.com/yukinarit/pyserde/blob/master/serde/compat.py#L21-L26